sssd_test_framework/markers.py

```diff
@@ -67,7 +77,13 @@ def pytest_runtest_setup(item: pytest.Item) -> None:
             # @pytest.mark.builtwith(client="passkey", ipa="passkey") ->
             # -> check if "passkey" is supported by both client and ipa
             requirements = dict(mark.kwargs)
+        elif len(mark.args) == 1 and mark.args:
```

Comment:
`elif len(mark.args) == 1 and isintance(mark.args[0], (Topology, KnownTopology, KnownTopologyGroup) and mark.kwargs`